x/execution/client/rest/query.go

```diff
@@ -58,7 +67,28 @@ func queryListHandlerFn(cliCtx context.CLIContext) http.HandlerFunc {
 			return
 		}
 
+		execs := make([]*execution.Execution, 0)
+		if err := cliCtx.Codec.UnmarshalJSON(res, &execs); err != nil {
+			rest.WriteErrorResponse(w, http.StatusInternalServerError, err.Error())
+			return
+		}
+
+		sortFunc := sort.Interface(execution.ByBlockHeight(execs))
+		for _, tag := range tags {
+			if tag == "sort='reverse'" {
+				sortFunc = sort.Reverse(sortFunc)
+			}
+		}
+		sort.Sort(sortFunc)
```

Comment:
I'm not sure why we need this customization. I would just force a descending order on the block height
```suggestion
		sort.Reverse(sort.Interface(execution.ByBlockHeight(execs)))
```

Reply:
I was testing what the tags were and implemented the reverse sort with it.
I can revert if it's not useful

Reply:
I would KISS especially as it is not a definitive code and will certainly be replaced when we have a better pagination system.